modules/telco-ran-ptp-operator.adoc

```diff
@@ -7,9 +7,9 @@
 = PTP Operator
 
 New in this release::
-* https://issues.redhat.com/browse/CNF-12504[Enable 2 Port Ordinary Clock Use Case for IA]
-* https://issues.redhat.com/browse/CNF-13189[CNF-13189: Remove the internal API and sidecar support]
-* https://issues.redhat.com/browse/CNF-16651[CNF-16651: 3 Card WPC T-G]
+* 2 Port Ordinary Clock Use Case for IA is enabled.
```

Comment:
Noting that there is no hyphen for 2 Port or 3 Card. Just wondering . . .is that the style you typically use in Telco docs for adjectives like these?